cfg/filebackend.go

```diff
@@ -47,10 +72,15 @@ func (fs *FileBackend) Save(config *Config) error {
 		os.MkdirAll(cfgDir, 0755)
 	}
 
-	j, err := json.MarshalIndent(config, "", "  ")
-	if err == nil {
-		err = ioutil.WriteFile(config.URL().Path, j, 0644)
+	var content []byte
+	var err error
+	if fs.format == FormatYAML {
+		content, err = yaml.Marshal(config)
+	} else {
+		content, err = json.MarshalIndent(config, "", "  ")
```

Comment:
@orsinium thanks, we really appreciate your contributions!
I guess omitting this `else` block for `json` would already work out as migration

Reply:
Sorry for the confusion. I know it's not hard to do, I'm just too careful and afraid to change the current default behavior :)

Also, it's not enough to change this line, be careful: this will rewrite `*.conf` file but the implementation in this PR always parses `*.conf` as JSON.